vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -36846,6 +36846,7 @@ unit:PERM_Metric_0DEG_C
   qudt:hasDimensionVector qkdv:A0E0L-1I0M0H0T1D0 ;
   qudt:hasQuantityKind quantitykind:Unknown ;
   qudt:iec61360Code "0112/2///62720#UAB294" ;
+  qudt:symbol "perm (0 ºC)" ;
```

Comment:
Here's another case where the degree symbol seems to have a weird line under it...